typescript/cosmos-sdk/src/registry/index.ts

```diff
@@ -0,0 +1,416 @@
+import * as types from '@hyperlane-xyz/cosmos-types';
+
+// amino converters can be null by default since the
+// converters for proto can be taken. In rare cases
+// they can differ, in that instance the amino
+// converters can be overridden here by adding the
+// methods "toJSON" and "fromJSON".
+export const REGISTRY = {
```

Comment:
Considering that this constant is exported from the cosmos SDK package, can we rename it to something like `COSMOS_MODULE_MESSAGE_REGISTRY` (or better name). Otherwise, it might be confusing with the actual hyperlane registry